src/controllers/check.your.answers.controller.ts

```diff
@@ -37,8 +37,8 @@ export const post = async (req: Request, res: Response, next: NextFunction) => {
     const overseaEntity: OverseasEntityCreated = await createOverseasEntity(req.session as Session, transaction.id as string);
     logger.info(`Overseas Entity Created, ID: ${overseaEntity.id}`);
 
-    const responseClosedTransaction = await closeTransaction(req.session as Session, transaction.id as string, overseaEntity.id);
-    logger.info(`Transaction Closed,  ${JSON.stringify(responseClosedTransaction)}`);
+    await closeTransaction(req.session as Session, transaction.id as string, overseaEntity.id);
+    logger.info(`Transaction Closed,  ${JSON.stringify(transaction.id)}`);
```

Comment:
could you put the word 'ID' in the message, just so we know what the number is,  maybe
`Transaction Closed, ID = ${JSON.stringify(transaction.id)}`

Reply:
actually, do you need the JSON.stringify?
could you use 
Transaction Closed, ID = ${transaction.id}